new-lamassu-admin/src/components/inputs/formik/Checkbox.js

```diff
@@ -5,7 +5,21 @@ import { Checkbox } from '../base'
 const CheckboxInput = memo(({ label, textAlign, fullWidth, ...props }) => {
   const { name, onChange, value } = props.field
 
-  return <Checkbox name={name} onChange={onChange} value={value} {...props} />
+  const settings = {
+    enabled: props.enabled ?? true,
```

Comment:
We should do 
```suggestion
    enabled: props.enabled ? true : false,
```
to consider the case when enabled is set to false

Reply:
This is perfectly correct, ignore my suggestion!